### ISSUE TITLE
[recurring events] event is displayed only once even if it would happen more than once on a single day

### ISSUE THREAD

**Issue by poljpocket** [ese-unibe-ch/ese2011-team3#127]
When a recurring event is spanning more than one day and still occurring when the next occurrence starts, the next occurrence isn't displayed.

This has to do with the algorithm for occurrences. When the algorithm finds an occurrence happening on a specific day, it stops and doesn't search on for other occurrences on the very same day of the very same event.

Maybe we will fix this in the future. But not now...
